src/electron/build.action.sh

```diff
@@ -29,17 +29,11 @@ for i in "$@"; do
     esac
 done
 
-npm run action src/www/build_electron -- \
-    --buildMode="${BUILD_MODE}"
+run_action src/www/build "${PLATFORM}" --buildMode="${BUILD_MODE}"
 
 WEBPACK_MODE="$(node scripts/get_webpack_mode.mjs --buildMode=${BUILD_MODE})"
 
 webpack \
-    --config=src/electron/electron_main.webpack.js \
+    --config=src/electron/webpack_electron_main.js \
```

Comment:
mm - so generally we prefer `[binary]_[platform]_[variant?]`

Reply:
I'd say so. It's a lot easier to navigate.